pkg/network/ovn_kubernetes.go

```diff
@@ -669,6 +671,18 @@ func validateOVNKubernetes(conf *operv1.NetworkSpec) []error {
 		if oc.GenevePort != nil && (*oc.GenevePort < 1 || *oc.GenevePort > 65535) {
 			out = append(out, errors.Errorf("invalid GenevePort %d", *oc.GenevePort))
 		}
+		if oc.V4InternalSubnet != "" {
```

Comment:
In the api doc, it says:
```
It must not overlap with any other subnet being used by OpenShift or by the node network. The size of the subnet must be larger than the number of nodes
```
should we validate the above or is it already validated somewhere else?

Reply:
I don't think we can validate the size of the cluster, as it's extendable after installation. But I can add the overlapping validation.